pytensor/link/pytorch/dispatch/scalar.py

```diff
@@ -38,3 +37,24 @@ def pytorch_func(*args):
             )
 
     return pytorch_func
+
+
+@pytorch_funcify.register(ScalarLoop)
+def pytorch_funicify_ScalarLoop(op, node, **kwargs):
+    update = pytorch_funcify(op.fgraph)
+
+    def inner(steps, start, constant, update=update, is_while=op.is_while):
```

Comment:
No need for optional kwargs, just access them inside the function from the scoping rules